module-services/service-cellular/connection-manager/ConnectionManagerCellularCommands.cpp

```diff
@@ -97,3 +97,11 @@ void ConnectionManagerCellularCommands::stopConnectionTimer()
 {
     cellular.connectionTimer.stop();
 }
+
+void ConnectionManagerCellularCommands::holdMinimumCpuFrequency()
+{
+    if (cellular.cpuSentinel) {
+        cellular.cpuSentinel->HoldMinimumFrequency(bsp::CpuFrequencyHz::Level_4);
+    }
+    vTaskDelay(200);
```

Comment:
Why is the delay needed?

Reply:
Because sentinel is working on messages and in 100 ms ticks.

Reply:
please at least add an explanation in comment.

Reply:
outdated